.gitignore

```diff
@@ -20,6 +20,9 @@
 /src/local_scheduler/format/*_generated.h
 /src/ray/gcs/format/*_generated.h
 
+# Ray results should be ignored
```

Comment:
Please remove. `ray_results` does not default to being written inside the `ray` directory (as far as I know?).